policy/modules/contrib/frr.te

```diff
@@ -0,0 +1,108 @@
+policy_module(frr, 1.0.0)
+
+########################################
+#
+# Declarations
+#
+
+type frr_t;
+type frr_exec_t;
+init_daemon_domain(frr_t, frr_exec_t)
+
+type frr_log_t;
+logging_log_file(frr_log_t)
+
+type frr_tmp_t;
+files_tmp_file(frr_tmp_t)
+
+type frr_lock_t;
+files_lock_file(frr_lock_t)
+
+type frr_conf_t;
+files_config_file(frr_conf_t)
+
+type frr_unit_file_t;
+systemd_unit_file(frr_unit_file_t)
```

Comment:
The private unit file type is not used anywhere else, is it supposed to happen so?

Reply:
@milosmalik  wrote about acceptance criteria of frr in bugzilla ticket, so I added this. And also when I look to /usr/lib/systemd/system/ is here frr.service file which is labeled as frr_unit_file_t.  But when was tested frr policy I didn find any AVC relating with frr_unit_file_t. So what do you think? Can I add some permission for frr_unit_file_t?